### ISSUE TITLE
Downloads from BBC failing

### ISSUE THREAD

**Issue by jfbauer432** [gpodder/gpodder#1518]
Downloads of one specific feed have been failing for a few weeks with a "403 Client Error: Forbidden for url" error.  It was working fine before that.  wget on same system is able to download it without issue.  I removed and re-added the feed, no change.

Output of download attempt with gpodder -v

`1687528461.561504 [gpodder.gtkui.main] DEBUG: Downloading episode: Human embryo models
1687528461.593952 [gpodder.my] DEBUG: Storing 1 episode download actions
1687528461.596470 [gpodder.download] INFO: 1 tasks to do, can start at most 1 threads, 0 threads currently running
1687528461.596778 [gpodder.download] INFO: Starting new worker thread.
1687528461.597097 [gpodder.util] DEBUG: run_in_background: <bound method DownloadQueueWorker.run of MainThread> (False)
1687528461.597819 [gpodder.download] INFO: Starting new thread: Thread-3 (run)
1687528461.623484 [gpodder.download] INFO: Thread-3 (run) is processing: Human embryo models
1687528461.623797 [gpodder.download] INFO: Downloading http://open.live.bbc.co.uk/mediaselector/6/redir/version/2.0/mediaset/audio-nondrm-download-low/proto/http/vpid/p0fwh07b.mp3
1687528461.625699 [urllib3.connectionpool] DEBUG: Starting new HTTP connection (1): open.live.bbc.co.uk:80
1687528462.134279 [urllib3.connectionpool] DEBUG: http://open.live.bbc.co.uk:80 "GET /mediaselector/6/redir/version/2.0/mediaset/audio-nondrm-download-low/proto/http/vpid/p0fwh07b.mp3 HTTP/1.1" 403 0
1687528462.135497 [gpodder.download] ERROR: HTTP 403 while downloading "Human embryo models": 403 Client Error: Forbidden for url: http://open.live.bbc.co.uk/mediaselector/6/redir/version/2.0/mediaset/audio-nondrm-download-low/proto/http/vpid/p0fwh07b.mp3
1687528462.136304 [gpodder.download] INFO: No more tasks for Thread-3 (run) to carry out.
1687528463.179179 [gpodder.gtkui.main] INFO: All tasks have finished.
1687528463.192117 [gpodder.gtkui.main] INFO: All tasks have finished.
`

Feed URL: https://podcasts.files.bbci.co.uk/p002vsnb.rss

gPodder 3.11.1 (2023-02-17)
Python 3.11.2
Ubuntu 23.04

**Comment by auouymous** [comment]
I was able to subscribe and download the "Human embryo models" episode without issue. Are you able to edit the `__init__.py` file and change the `user_agent` line to `user_agent = 'Wget/1.21.4`? If downloading works after restarting gpodder then bbc might be blocking your IP with the gpodder UA.

```python
user_agent = 'gPodder/%s (+%s) %s' % (__version__, __url__, platform.system())  
user_agent = 'Wget/1.21.4'
```

**Comment by jfbauer432** [comment]
Changing the user_agent didn't fit it, but I stated digging deeper.  Discovered that gpodder was sending an 'Authorization: Basic' header, which from what I could see in the gpodder code, it should not have been doing (auth was set to None in the session.get() call).  Digging through the python requests documentation I stumbled upon references to netrc.  Then I realized what it probably was.  I had an ancient .netrc file that had a default entry for anonymous FTP.  Removing that fix it.
